.yamato/all-universal.yml

```diff
@@ -6,17 +6,15 @@
 All_Universal_trunk:
     name: All Universal CI - trunk
     dependencies:
-      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_Standalone_trunk
+      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_Standalone_mono_2_Linear_trunk
          rerun: on-new-revision
-      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_editmode_trunk
+      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_editmode_mono_2_Linear_trunk
          rerun: on-new-revision
-      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_playmode_trunk
+      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_playmode_mono_2_Linear_trunk
          rerun: on-new-revision
-      -  path: .yamato/universal-win-dx11.yml#Universal_Win_DX11_playmode_XR_trunk
```

Comment:
I think `Universal_Win_DX11_playmode_XR_trunk` is missing, is it wanted? Same for 2020.2 and custom-revision

Reply:
accidental i think, checking